### ISSUE TITLE
Supporting non english characters

### ISSUE THREAD

**Issue by erezo9** [mattermost/mattermost-plugin-memes#44]
Hi,
First off, Great plugin
have a question, hebrew is not supported, all i get when i try to put hebrew characters, the pic comes blank
is that something that can work?

**Comment by hanzei** [comment]
@erezo9 Could this be a duplicate of https://github.com/mattermost/mattermost-plugin-memes/issues/14?

**Comment by erezo9** [comment]
@hanzei no, because i dont have symbols or something like that, only blank message
try using the meme 
/meme picard-wtf ניסוי
the string at the end is hebrew, maybe you can debug and see what happenes
